client_plugin/drivers/shared/dockerops/dockerops.go

```diff
@@ -65,14 +66,24 @@ const (
 	checkDuration = 5 * time.Second
 	// Time between successive checks for deleting a volume
 	checkSleepDuration = time.Second
-	// Timeout to mark Samba service launch as unsuccessful
-	sambaRequestTimeout = 30 * time.Second
+	// default Timeout to mark Samba service launch as unsuccessful
+	defaultServiceStartTimeOutInSecond = 30
```

Comment:
nit: slightly shorter name (e.g. _defaultSvcStartTimeoutSec_ or even _defaultStartTimeout_ with comment) would be nice.